docs/aerodromes/classc/EastSale.md

```diff
@@ -4,7 +4,149 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+| Name               | Callsign           | Frequency        | Login Identifier             |
+| ------------------ | --------------     | ---------------- | -----------------------------|
+| East Sale ADC      | East Sale Tower    | 118.300          | ES_TWR                       |
+| East Sale SMC      | East Sale Ground   | 127.250          | ES_GND                       |
+| East Sale ACD      | East Sale Delivery | 134.100          | ES_DEL                       |
+| East Sale ATIS     |                    | 125.40           | YMES_ATIS                    |
+
+## Airspace
+ES ADC owns the Class C airspace from `SFC` to `A030`, within a 5nm radius of the ESL ARP. This airspace is designed to facilitate helicopter and circuit traffic.
+
+## Flight Category
+- Aircraft operating within ESL military airspace must be VFR unless otherwise indicated via flight plan or the pilot requests IFR or SVFR, due to weather.
+- Local area IFR operations should be minimised to those necessary, and aircraft must remain VFR until specifically cleared IFR by ATC. IFR clearance is a readback requirement.
+- **IFR Clearance Procedures**:
+  - **Prior to Departure**:  
+    Pilots must request IFR clearance during their initial clearance request or during taxi. ATC will clear the aircraft IFR as part of the initial airways clearance. Pilots are not required to use AIP phraseology when upgrading to IFR in this situation.
+  
+  - **Airborne**:  
+    If the pilot is already airborne and needs to switch to IFR, they must request a change of category as per the Local RTF Procedures. ATC will clear the aircraft as soon as practicable. If the clearance is not immediately available, ATC will instruct the pilot to "REMAIN VFR" and provide necessary control instructions or expectations.
+  
+  - **IFR Climb**:  
+    On departure from ESL or WSL, a pilot may request an IFR climb to penetrate weather en route to a training area. During the day, unless being vectored, the pilot is responsible for terrain clearance below LSALT. Aircraft cleared for an "IFR CLIMB" must continue climbing, not above their cleared level, and report "CANCEL IFR" when able to resume VFR.
+  
+- **Resuming VFR**:
+  - Recovering military aircraft must automatically revert to VFR at the following points:
+    - At the initial point when recovering via military stream landing pattern (initial and pitch).
+    - At Hi-Key.
```

Comment:
Might be overkill this section. Hi-Key I guess is for flameout/practice forced landings situations

Reply:
If we're gonna leave it in (which I'm open to), we'll have to define what it is, either through the abbreviations page, or with a !!! note